meta-mbl-distro/scripts/create-update-payload

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""
+This script will create a bootloader update payload tar archive.
+
+You can specify a compression type for the tar archive. The default is gz and
+the available options are gz, bz2 and xz.
+
+To use the script you can either pass in a space separated list of file names
+or a list of bootloader slot numbers. We then map the file names of the update
+components to the corresponding bitbake variables, which are named according
+to the bootloader slot the file will end up in.
+
+We rename the files MBL_WKS_BOOTLOADER[n] so the update scripts on the target
+only have to know which bootloader slot they're updating, and are generic for
+any MACHINE target.
+
+The script assumes the files you want to archive are in DEPLOY_DIR_IMAGE in
+your MBL workspace.
+
+The script supports multi-component or single component payloads.
+
+Example invocation:
+
+create-update-payload --bootloader-slots 1 2 --output-tar-path tmp/payload.tar
```

Comment:
How does the script know where to find the bootloader files? Do I have to be in a particular directory to run this command? Or do I have to have sourced `setup-environment`?

Reply:
The script looks for the files in `DEPLOY_DIR_IMAGE` so I think we need to have sourced `setup-environment`